app/javascript/controllers/scrollable_controller.js

```diff
@@ -10,7 +10,7 @@ export default class extends Controller {
   checkScroll() {
     const target = this.scrollableTarget
     const isAtTop = target.scrollTop === 0
-    const isAtBottom = target.scrollHeight - target.scrollTop === target.clientHeight
+    const isAtBottom = Math.round(target.scrollHeight - target.scrollTop) === target.clientHeight
```

Comment:
Oddly, I have one conversation where the scrollHeight is `xxx.5` ! I've never run into fractional pixel heights but rounding solved it